src/components/LeftDrawer/LeftDrawer.tsx

```diff
@@ -8,6 +8,7 @@ import SettingsIcon from 'assets/svgs/settings.svg?react';
 import TalawaLogo from 'assets/svgs/talawa.svg?react';
 import styles from 'style/app.module.css';
 import useLocalStorage from 'utils/useLocalstorage';
+import ProfileDropdown from 'components/ProfileDropdown/ProfileDropdown';
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with exceptions for package based CSS.
1. Delete the CSS file it currently uses.
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296